test/test_http.py

```diff
@@ -60,9 +63,13 @@ def is_uptime(value):
     return True
 
 
+@pytest.mark.parametrize("api", ["cli", "dbus"])
 @have_json
-def test_info_headers(create_system_files, system, http_server):
+def test_info_headers(create_system_files, system, http_server, api):
```

Comment:
This is only indirectly related to the patch series, isn't it?

Reply:
The DBus and CLI code paths for info are slightly different, so we should test them both.

Reply:
With the last test suite fixes, the patch coverage looks good now.